### PR TITLE
fix: add missing labels in standard metrics

### PR DESCRIPTION
**Pull request by Xunzhuo**
Refer to: https://github.com/istio/proxy/blob/master/extensions/common/istio_dimensions.h#L27-L52

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
Waiting on an approval from @istio/wg-user-experience-maintainers as they own the page.

**Review by douglas-reid — CHANGES_REQUESTED** [review]
*Downstream IP* is definitely not a standard label for Istio telemetry.

**Comment by douglas-reid** [comment]
I think this PR is confusion dimensions in stackdriver access logs with dimensions exposed by standard metrics.

File: content/en/docs/reference/config/metrics/index.md
Comment (douglas-reid) [inline content/en/docs/reference/config/metrics/index.md:125]:
```suggestion
```

**Comment by Xunzhuo** [comment]
Thanks @douglas-reid

**Review by douglas-reid — APPROVED** [review]

**Pull request merged**
